internal/stat/stat_test.go

```diff
@@ -13,30 +12,31 @@ import (
 )
 
 func TestGetBlockSize(t *testing.T) {
-	s, err := GetBlockSize(os.DevNull)
+	size, err := GetBlockSize(os.DevNull)
 	require.NoError(t, err)
 
-	if s <= 0 {
-		t.Fatalf("invalid disk block size: %d, must be greater than 0", s)
-	}
+	require.Positive(t, size)
+}
+
+func TestGetBlockSizeFromCurrentFS(t *testing.T) {
+	size, err := GetBlockSize(".")
+
+	require.NoError(t, err)
+	require.Positive(t, size)
 }
 
 func TestGetFileAllocSize(t *testing.T) {
-	const size = 4096
+	const expectedMinAllocSize = 512
```

Comment:
Minimum block size across platforms / file systems.
Most systems will use a multiple of this value, usually powers of 2 such as, 1K, 2K, 4K, 8K, 16K and so on.